appsembleredx/signals.py

```diff
@@ -21,26 +22,31 @@
 from xmodule.contentstore.content import StaticContent
 from django.core.files.storage import get_storage_class
 
-from course_modes.models import CourseMode, CourseModeExpirationConfig
+from course_modes.models import CourseMode
 from certificates import models as cert_models
 
-from appsembleredx.app_settings import (
-    DEFAULT_COURSE_MODE_SLUG, 
-    mode_name_from_slug, 
-    USE_OPEN_ENDED_CERTS_DEFAULTS,
-    ALWAYS_ENABLE_SELF_GENERATED_CERTS,
-    DISABLE_SELF_GENERATED_CERTS_FOR_SELF_PACED,
-    DEFAULT_CERT_SIGNATORIES,
-    ACTIVATE_DEFAULT_CERTS,
-    DISABLE_COURSE_COMPLETION_BADGES
-)
+from appsembleredx import app_settings
 
 DEFAULT_CERT = """
     {{"course_title": "", "name": "Default", "is_active": {},
     "signatories": {}, "version": 1, "editing": false,
     "description": "Default certificate"}}
 """
 
+logger = logging.getLogger(__name__)
+
+
+def disable_if_certs_feature_off(func):
+    @wraps(func)
+    def noop_handler(*args, **kwargs):
+        logger.warn('{} signal handler disabled since CERTIFICATES_ENABLED is False'.format(func.__name__))
+
+    if settings.FEATURES.get('CERTIFICATES_ENABLED', False):
```

Comment:
@tkeemon Yeah I need to revisit this.  I realized afterward that `CERTIFICATES_ENABLED` isn't the best test since it's exclusively used for PDF certs.  We just were still setting it to True for any customer using certs even though everyone has moved to HTML.

Reply:
This repo has a successor post-Ficus anyhow, `appsembler_credentials_extensions`.